components/styles.js

```diff
@@ -100,6 +102,19 @@ const styles = StyleSheet.create({
   weekendDayText: {
     color: 'black',
   },
+  selectedRangeBarWrapper: {
+    flexDirection: 'row',
+  },
+  selectedRangeBar: {
+    width: DEVICE_WIDTH / 14,
+    height: 28,
+    backgroundColor: '#F22169',
+  },
+  emptyRangeBar: {
```

Comment:
let's also add the option for the half transparent pink in the filtering calendar.